Cargo.toml

```diff
@@ -1,30 +1,37 @@
 [workspace]
 
 members = [
-    "platform/api",
-    "platform/image_processor",
-    "video/edge",
-    "video/ingest",
-    "video/transcoder",
-    "video/lib/*",
-    "video/api",
-    "video/player",
-    "video/player_types",
-    "video/common",
-    "video/cli",
-    "binary-helper",
-    "utils",
-    "proto",
-    "config",
-    "config/derive",
+    # "platform/api",
+    "image-processor",
+    "image-processor/proto",
+    # "video/edge",
+    # "video/ingest",
+    # "video/transcoder",
+    # "video/lib/*",
+    # "video/api",
+    # "video/player",
+    # "video/player_types",
+    # "video/common",
+    # "video/cli",
+    # "binary-helper",
+    # "utils",
+    # "proto",
+    # "config",
+    # "config/derive",
```

Comment:
whats going on here?

Reply:
due to changes the other libs dont build, and we havent fixed them so in order to use i commented them out